src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1640,15 +1668,19 @@ protected function get_attendees_by_order_id( $order_id, $ticket_id = null ) {
 		}
 
 		$find_by_args = [
-			'post_id'   => $order_id,
-			'ticket_id' => [],
+			'post_id'        => $order_id,
+			'ticket_id'      => [],
+			'posts_per_page' => 1,
 		];
 
 		if ( $ticket_id ) {
 			$find_by_args['ticket_id'] = (array) $ticket_id;
 		}
 
-		$orders = Tribe__Tickets__Commerce__PayPal__Order::find_by( $find_by_args );
+		$orders = Tribe__Tickets__Commerce__PayPal__Order::find_by( $find_by_args, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: cf0605cccdf7338b2bbb7be497e10d5a</sub>